tslint.json

```diff
@@ -20,6 +19,7 @@
 		"arrow-parens": [true, "ban-single-arg-parens"],
 		"only-arrow-functions": false,
 		"max-classes-per-file": false,
-		"no-bitwise": false
+		"no-bitwise": false,
+		"callable-types": false
```

Comment:
Why does this rule need to be disabled?

Reply:
I'm not sure, I added it back and everything still works fine.